.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: ci
+
+on:
+  push:
+
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    container:
+      image: nat1405/niri:0.1
```

Comment:
I'm not sure how this is supposed to work. You test an image with a set version before you deploy the same image? Or is `0.1` meant to be `latest`, i.e moving head?

Reply:
Good point, `0.1` is meant to be a moving head. Maybe it makes more sense to run some sort of `dev` container for the tests, then use it as a base image to build and deploy a `production` container with the relevant version of this python package installed. This is good, because it separates the minimum logic necessary to run & test the container from the `Arcade` (or `skaha`) specific deployment steps. Do you think it's worth the dev effort and that that would keep things more clear? (I have less experience with seeing the long-term maintainability/clear communication picture)...